projects/angular/src/emphasis/alert/utils/alert-types.ts

```diff
@@ -4,5 +4,9 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-// @TODO Make this an enum
-export const ALERT_TYPES: string[] = ['info', 'warning', 'danger', 'success'];
+export enum AlertType {
```

Comment:
My suggestion here is to turn this into

`export type AlertType = 'info' | 'warning' | 'danger' | 'success';`

While the enum is a good data structure to iterate over and keep the type checker protecting you enforcing people to provide it in the component where they are declaring an Alert it will be too much. 

Also this will cause a breaking change if we provide it as a constrain when providing the AlertType, which we do in this PR.

Reply:
I believe we can keep the enum but also provide a `type` and reuse the enum internally but not enforcing the users to use it.

Reply:
[TypeScript playground for reference](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcCCAbYsAqzUC8cA5AJYB2AZhMXAD4kDuAhlORQOa0PEAmz5Dtm4kAzgFcAxpOCjRxANwAoJaEiw4kiOVHxg5cQFt0WXPgCiB4wG8lce-YCSVCHCJkXxADR2HcAOqs7IJuTEGc3r4OACICQlChfHHCPn5wAMpSMnKJEtKy8j4AvipaOvCiwDCY2DB4KKEAFMymdfgAXCa19cAAlG4AfHC2SiVKldWtPY01ZiiWRgB0sYLYvQpAA)